components/account/AccountMoreButton.vue

```diff
@@ -62,6 +68,23 @@ const toggleReblogs = async () => {
 
       <template v-if="currentUser">
         <template v-if="!isSelf">
+          <template v-if="relationship!.following">
```

Comment:
Agreed.  I went back and forth - have seen it both ways.  This would resolve the long text issues.  I am not normally up this late - so will pic this back up tomorrow (today?).  

Question - when the automated testing ran, I saw some Typescript errors and sure enough those Mastodon API [notify](https://docs.joinmastodon.org/methods/accounts/#form-data-parameters-2) and [notifying](https://docs.joinmastodon.org/methods/accounts/#200-ok-9) fields are not included in the Masto.js [FollowAccountParam](https://neet.github.io/masto.js/interfaces/mastodon.v1.FollowAccountParams.html) and [Relationship](https://neet.github.io/masto.js/interfaces/mastodon.v1.Relationship.html) entities.  

It worked regardless but Typescript isn't happy about it of course.  If that's the case are we on hold until masto.js adds those into their codebase?  I can't believe it doesn't exist.

Reply:
It's an issue with masto.js. Submitted a PR https://github.com/neet/masto.js/pull/811

We can use `// @ts-expect-error ` as a temporal solution.

Reply:
Made those changes, but since the public release, I couldn't commit back to the branch. I created a new branch with a new pull request to merge into this one.... apologies if this wasn't the right procedure!

https://github.com/elk-zone/elk/pull/1045

Reply:
Thank you!